cypress/support/commands.js

```diff
@@ -0,0 +1,49 @@
+/// <reference types="Cypress" />
+
+// ***********************************************
+// This example commands.js shows you how to
+// create various custom commands and overwrite
+// existing commands.
+//
+// For more comprehensive examples of custom
+// commands please read more here:
+// https://on.cypress.io/custom-commands
+// ***********************************************
+//
+//
+// -- This is a parent command --
+// Cypress.Commands.add("login", (email, password) => { ... })
+//
+//
+// -- This is a child command --
+// Cypress.Commands.add("drag", { prevSubject: 'element'}, (subject, options) => { ... })
+//
+//
+// -- This is a dual command --
+// Cypress.Commands.add("dismiss", { prevSubject: 'optional'}, (subject, options) => { ... })
+//
+//
+// -- This is will overwrite an existing command --
+// Cypress.Commands.overwrite("visit", (originalFn, url, options) => { ... })
+
+Cypress.Commands.add('login', (username, password) => {
+    cy.request({
+        method: 'POST',
+        url: 'http://localhost:8080/dhis-web-commons-security/login.action',
+        body: {
+            j_username: username,
+            j_password: password,
+        },
+        form: true,
+        log: true,
+    });
+});
+
+Cypress.Commands.add('persistLogin', () => {
+    Cypress.Cookies.preserveOnce('JSESSIONID');
+});
+
+Cypress.Commands.add('loadPage', () => {
+    cy.visit('http://localhost:8082');
```

Comment:
Maybe this url can go into a config file somewhere?

Reply:
I think it should be an environment variable, actually, but yeah it should be configurable.